todo_app/app.py

```diff
@@ -1,24 +1,51 @@
 from flask import Flask, redirect, render_template, request
-from todo_app.data.trello_items import get_items, add_item, complete_item
-
+from todo_app.data.trello_items import get_items, add_item_to_list, complete_item
 from todo_app.flask_config import Config
+import os
+
+def create_app():
+  app = Flask(__name__)
+  app.config.from_object(Config())
+
+  trello_base = "https://trello.com/1/"
+  board_id = os.environ.get("TRELLO_BOARD_ID")
+  trello_todo_list_id = os.environ.get('TRELLO_TODO_LIST_ID')
+  trello_done_list_id = os.environ.get('TRELLO_DONE_LIST_ID')
+  trello_keys = {
+    'key': os.environ.get('TRELLO_KEY'),
+    'token': os.environ.get('TRELLO_TOKEN')
+  }
+
+  @app.route('/')
+  def get_todos():
+      items = get_items(board_id, trello_base, trello_keys)
+      item_view_model = ViewModel(items)
+      return render_template('index.html', view_model = item_view_model)
 
-app = Flask(__name__)
-app.config.from_object(Config())
+  @app.route('/add-todo', methods=['POST'])
+  def add_todo():
+      newTodo = request.form.get('new-todo')
+      add_item_to_list(newTodo, trello_base, trello_keys, trello_todo_list_id)
+      return redirect('/')
 
+  @app.route('/complete-todo/<id>')
+  def complete_todo(id):
+      complete_item(id, trello_base, trello_keys, trello_done_list_id)    
+      return redirect('/')
+  return app
 
-@app.route('/')
-def get_todos():
-    [todo_items, done_items] = get_items()
-    return render_template('index.html', todo_items = todo_items, done_items=done_items)
+class ViewModel:
```

Comment:
Generally, each class should go in a separate file, and the same is true for the `Item` class which is in `trello_items.py`. This is a good practice because it will keep the codebase more organized, even if the class has only a few lines of code